rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -113,8 +113,14 @@ gs_frame::gs_frame(QScreen* screen, const QRect& geometry, const QIcon& appIcon,
 	setScreen(screen);
 	setGeometry(geometry);
 	setTitle(qstr(m_window_title));
-	setVisibility(startup_visibility);
-	create();
+
+	if (g_cfg.video.renderer != video_renderer::opengl)
+	{
+		// Do not display the window before OpenGL is configured!
+		// This works fine in windows and X11 but wayland-egl will crash later.
+		setVisibility(startup_visibility);
```

Comment:
Doesn't this kinda break the "Windowed" workaround for wayland?

Reply:
It works fine from my testing, but you can't set visibility either without making egl upset. I'll double-check this on my main rig later to confirm.